packages/expo/src/environment/logging.fx.ts

```diff
@@ -2,6 +2,7 @@ import Constants from 'expo-constants';
 
 import Logs from '../logs/Logs';
 import RemoteLogging from '../logs/RemoteLogging';
+import './muteCyclicWarnings.fx';
```

Comment:
Move the side effect import to the top of this file: https://github.com/expo/expo/blob/master/guides/Expo%20JavaScript%20Style%20Guide.md#imports